src/silvimetric/commands/info.py

```diff
@@ -1,37 +1,57 @@
-import json
+import numpy as np
+from uuid import UUID
 
-from ..resources import Storage
+from ..resources import Storage, Bounds
 
-def info(tdb_dir, history):
+def check_values(start_time, end_time, bounds, name):
+    if start_time is not None and not isinstance(start_time, np.datetime64):
+        raise TypeError(f'Incorrect type of "start_time" argument.')
+    if end_time is not None and not isinstance(end_time, np.datetime64):
+        raise TypeError(f'Incorrect type of "end_time" argument.')
+    if bounds is not None and not isinstance(bounds, Bounds):
+        raise TypeError(f'Incorrect type of "bounds" argument.')
+    if name is not None:
```

Comment:
Please add a comment here. Why do we need a name and why are we making a uuid?

Reply:
We don't necessarily need a name, but if you do provide a name, it needs to be a string that's parsable by the `UUID` class